requirements/requirements.txt

```diff
@@ -44,79 +44,81 @@ django-taggit==0.22.2
 django==1.11.20
 djangorestframework-yaml==1.0.3
 djangorestframework==3.7.7
+docutils==0.14            # via python-daemon
 future==0.16.0            # via django-radius
-hyperlink==18.0.0         # via twisted
-idna==2.8                 # via hyperlink, requests
+hyperlink==19.0.0         # via twisted
+idna==2.8                 # via hyperlink, requests, twisted
 incremental==17.5.0       # via twisted
 inflect==2.1.0            # via jaraco.itertools
 irc==16.2
 isodate==0.6.0            # via msrest, python3-saml
 jaraco.classes==2.0       # via jaraco.collections
-jaraco.collections==2.0   # via irc, jaraco.text
+jaraco.collections==2.0   # via irc
 jaraco.functools==2.0     # via irc, jaraco.text, tempora
-jaraco.itertools==4.4.1   # via irc
+jaraco.itertools==4.4.2   # via irc
 jaraco.logging==2.0       # via irc
 jaraco.stream==2.0        # via irc
-jaraco.text==2.0          # via irc, jaraco.collections
+jaraco.text==3.0          # via irc, jaraco.collections
 jinja2==2.10.1
 jsonpickle==1.1           # via asgi-amqp
 jsonschema==2.6.0
-kombu==4.2.1              # via asgi-amqp, celery
-lxml==4.3.1               # via xmlsec
+kombu==4.5.0              # via asgi-amqp, celery
+lockfile==0.12.2          # via python-daemon
```

Comment:
Hmm, maybe I missed this dependency of a dependency. python-daemon version didn't change, but this is being pulled in now